requirements/dist-build-constraints.in

```
@@ -6,7 +6,6 @@ setuptools >= 34.4
 
 # Plugins
 setuptools_scm[toml] >= 3.5
```

Make it match `pyproject.toml`:
```suggestion
 setuptools_scm[toml] >= 7.0.0
```